sidecar/test/figwheel_sidecar/build_middleware/file_reloading_test.clj

```diff
@@ -0,0 +1,22 @@
+(ns figwheel-sidecar.build-middleware.file-reloading-test
+  (:require [clojure.java.io :as io]
+            [figwheel-sidecar.build-middleware.javascript-reloading :as js-reloading]
+            [clojure.test :as test :refer [deftest is testing]]))
+
+(deftest javascript-file-locations
+  (let [output-dir "/home/arichiardi/.boot/cache/tmp/home/arichiardi/git/ekg/9h3/-972l6l/scripts/main.out"
+        file-string "/home/arichiardi/.boot/cache/tmp/home/arichiardi/git/ekg/9h3/-boccbb/scripts/test.js-boccbb/scripts/test.js"]
```

Comment:
Also, they say that you should never write your own test, please feel free to break it 😀😀